schema/spacewalk/common/tables/rhnServerPackage.sql

```diff
@@ -38,5 +38,15 @@ CREATE UNIQUE INDEX rhn_sp_snep_uq
     
     ;
 
+CREATE INDEX rhn_server_package_nid_evrid_idx 
+    ON rhnServerPackage (name_id, evr_id)
```

Comment:
Do we search for name_id and EVR_id only on this table? If we always use the server_id on the queries I already see an index with "(server_id, name_id, evr_id, package_arch_id)".